2024-04-17 18:49:26,245 - Log.py[line:36] - ERROR: USB device 12345678 is offline
